src/pages/public_registration/LogPanel.tsx

```diff
@@ -109,31 +114,101 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '0.5rem' }}>
       {registration && (
         <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
-          <Box sx={{ display: 'flex', flexDirection: 'column' }}>
-            <Typography>{t('common.created')}:</Typography>
-            {organizationQuery.isLoading || userQuery.isLoading ? (
-              <Skeleton sx={{ width: '4rem' }} />
-            ) : (
-              <Typography>
-                {organizationQuery.data ? organizationQuery.data?.acronym : t('common.unknown')}
-                {userQuery.data ? `, ${getFullName(userQuery.data.givenName, userQuery.data.familyName)}` : ''}
-              </Typography>
-            )}
-          </Box>
-          <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          <Typography>{t('common.created')}</Typography>
+          <Tooltip title={new Date(registration.createdDate).toLocaleTimeString()} enterDelay={tooltipDelay}>
+            <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          </Tooltip>
+          {organizationQuery.isLoading || userQuery.isLoading ? (
+            <Skeleton sx={{ width: '4rem' }} />
+          ) : (
+            <Typography>
+              {organizationQuery.data ? organizationQuery.data?.acronym : t('common.unknown')}
+              {userQuery.data ? `, ${getFullName(userQuery.data.givenName, userQuery.data.familyName)}` : ''}
+            </Typography>
+          )}
         </StyledStatusMessageBox>
       )}
       {logs
         .sort((a, b) => new Date(a.modifiedDate).getTime() - new Date(b.modifiedDate).getTime())
         .map((logItem, index) => {
           const bgColor = logItem.type === 'PublishingRequest' ? 'publishingRequest.main' : 'doiRequest.main';
+          const modifiedDate = new Date(logItem.modifiedDate);
           return (
             <StyledStatusMessageBox key={index} sx={{ bgcolor: `${bgColor}` }}>
```

Comment:
```suggestion
            <StyledStatusMessageBox key={index} sx={{ bgcolor: bgColor }}>
```

fyi, så har vi en annen løsning for å få riktig farge i `TicketListItem.tsx`, som ser ca slik ut: `color: ticketColor[ticket.type]`. Syns ikke den ene er mye bedre enn den andre, så opp til deg om du vil gjøre det sånn eller sånn :)

Reply:
Liker den bedre! Fjerner en linje kode, og tar i tillegg høyde for flere ticket types om det skulle implementeres senere 🙌